src/transport_ofi.c

```diff
@@ -387,13 +415,17 @@ static inline void exchange_and_av_insert(const int npes)
             0,
 	    NULL);
     if(ret!=npes){
-	RAISE_ERROR(ret);
-    }
+	OFI_ABORT("av insert failed\n");
+	return ret;
+    } else
+	ret = 0;
 
     free(alladdrs);
+
+    return ret;
```

Comment:
Would it be cleaner to return 0 here?  Or is there an instance where you can reach this statement and have nonzero ret?  This would allow you to delete "else ret = 0" above.